### PR TITLE
fixes #29, Add option to associate multiple VPCs to service discovery namespace

### LINKED ISSUE #29: Add functionality to associate multiple VPC to Service discovery namespace
_As a SysOps_
_I want to have the capability to associate multiple VPC to Service discovery namespace_
_So that I can access service discovery from multiple VPCs_

### PR DESCRIPTION
**Pull request by pradeepbhadani**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,15 +16,16 @@ For more information please refer to the main [Apiary](https://github.com/Expedi
 | graphite_port | Graphite server port. | string | `2003` | no |
 | graphite_prefix | Prefix addded to all metrics sent to Graphite from this Waggle Dance instance. | string | `waggle-dance` | no |
 | ingress_cidr | Generally allowed ingress CIDR list. | list | - | yes |
-| instance_count | Number of ECS task to create. | string | `1` | no |
+| instance_count | Number of ECS tasks to create. | string | `1` | no |
 | instance_name | Waggle Dance instance name to identify resources in multi-instance deployments. | string | `` | no |
 | local_metastores | List of federated Metastores in current account. | list | `<list>` | no |
-| memory | The amount of memory (in MiB) used by Waggle Dance task. Valid values: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-cpu-memory-error.html | string | `4096` | no |
+| memory | The amount of memory (in MiB) used to allocate for Waggle Dance container. Valid values: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-cpu-memory-error.html | string | `4096` | no |
```
Comment (massdosage) [inline README.md]: **the** Waggle Dance container

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,3 +7,5 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 
 ### Added
 - Option to register route53 cnames for vpc endpoints - see [#20](https://github.com/ExpediaInc/apiary-waggledance/issues/20).
+- Option to associate multiple VPCs to service discover namespace - see
```
[the remark this answers is not in the document]
Comment (pradeepbhadani) [reply CHANGELOG.md]: updated

File: README.md (hunk 1 above)
Comment (pradeepbhadani) [reply README.md]: updated

File: main.tf [hunk 3]
```diff
@@ -180,3 +180,9 @@ resource "aws_service_discovery_service" "metastore_proxy" {
     failure_threshold = 1
   }
 }
+
+resource "aws_route53_zone_association" "secondary" {
```
Comment (rpoluri) [inline main.tf]: I think we also need specify vpc_region
https://www.terraform.io/docs/providers/aws/r/route53_zone_association.html#vpc_region
current code may not work when deploying to non default regions.

Comment (pradeepbhadani) [reply main.tf]: But we will always set default region when declaring AWS_PROFILE?
But its good to put the region to make sure code doesn't break. Updating PR

**Pull request merged**